darksky.yaml

```diff
@@ -0,0 +1,125 @@
+    description:  Interface to communicate to the Dark Sky weather service
+schemes:
+  # the Dark Sky REST API is served over HTTPS.
+  - https
+# format of the responses to the client (Accepts)
+produces:
+  - application/json
+
+################################################################################
+#                                           Paths                              #
+################################################################################
+paths:
+  /{auth_token}/{lat_and_long_or_time}:
+    get:
+      tags:
+        - calls
+      operationId:  forecast
+      summary:      Make a request to Dark Sky
+      description:  Fetch a weather forecast or get historical weather data based of input latitude and longitude
+      x-exosite-example:    |
+                    --#ENDPOINT GET /darksky/weather
+
+                    weather_data = Darksky.forecast({auth_token = 'db064d611dc600000000000000000000', lat_and_long_or_time = '44.977753,93.265011'})
+
+                    --lat and long and time string with ISO 8601 data string
+                    -- weather_data = Darksky.forecast({auth_token = 'db064d611dc600000000000000000000',lat_and_long_or_time = '44.977753,93.265011,2017-10-26T18:53:08+00:00'})
+                    --lat and long and time string with UNIX timestamp
+                    -- weather_data = Darksky.forecast({auth_token = 'db064d611dc600000000000000000000',lat_and_long_or_time = '44.977753,93.265011,1509047567'})
+
+                    -- Service call with some optional request parameters
+                    --[[ weather_data = Darksky.forecast({auth_token = 'db064d611dc600000000000000000000',
+                                                        lat_and_long_or_time = '44.977753,93.265011,1509047567',
+                                                        exclude = 'flags',
+                                                        lang = 'de'
+                    })]]
+
+                    print(to_json(weather_data))
+
+                    response.message = weather_data
+      parameters:
+        - name:         auth_token
+          in:           path
+          description:  Dark Sky Authentication Token
+          type:         string
+          required:     true
+        - name:         lat_and_long_or_time
+          in:           path
+          description:  A single comma delimited string containing Latitude and Longitude information with possibly a UNIX timestamp or ISO 8601 date string. 
+          type:         string
+          required:     true
+        - name:         exclude
+          in:           query
+          description:  Exclude some keys from the Dark Sky forecast response
+          type:         string
+        - name:         extend
+          in:           query
+          description:  "Fetch the next 168(7 days) worth of data, hour-by-hour, instead of the next 24. WARNING: Could take a long time and cause your Murano script to fail."
```

Comment:
"Fetch the next 168 **hours**(7 days) worth of data..."

Reply:
Resolved!